app/views/api/v3/analytics/user_analytics/_progress_cohort_report.html.erb

```diff
@@ -30,15 +30,15 @@
     <% @user_analytics.statistics.dig(:cohorts).each do |cohort| %>
       <div class="card cohort">
         <h6>
-          <%= cohort[:registered] %> <%= raw t("analytics.patient", count: cohort[:registered]) %>
+          <%= registered_patients = cohort[:registered] || 0 %> <%= raw t("analytics.patient", count: registered_patients) %>
```

Comment:
The first ERB tags should be `<% %>`. We don't want/need to print the result the assignment to the page.

Reply:
Do we want to use a helper method like this one? https://github.com/simpledotorg/simple-server/blob/master/app/helpers/dashboard_helper.rb#L7

Reply:
Or one step further, do we want to do this defaulting straight at the source, inside `UserAnalytics`?

Reply:
>The first ERB tags should be <% %>. We don't want/need to print the result the assignment to the page.

We need this `<%= %>` because we are showing this number on the page. 

Also, this view will be deleted when we release the new progress tab. Shall we just keep it this way?